tests/unit/test_driven.py

```diff
@@ -16,13 +16,11 @@
 import pandas as pd
 import pytest
 
-from model import storage
+from model.driven import adjust_fluxes2model
 
 
-def test_driven():
-    from model.driven import adjust_fluxes2model
-
-    model = storage.get('iJO1366').model
+@pytest.mark.skip(reason="Does not seem to return deterministic flux distributions, must be looked into")
```

Comment:
With larger metabolic models there are often multiple possible variable values that lead to the same objective value. It's not communicated in the measurements but the distance, i.e., the objective value must be stable. You could then add up the absolute values of the differences between the measured fluxes and the ones returned from `adjust_fluxes2model`. That sum should be approximately the objective value.

Reply:
Thanks, that makes more sense to test for and does work much better. I've adjusted the test accordingly.